base/src/main/java/org/aya/core/visitor/TermPostMap.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020-2022 Yinsen (Tesla) Zhang.
+// Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
+package org.aya.core.visitor;
+
+import org.aya.core.term.Term;
+
+import java.util.function.Function;
+
+public record TermPostMap(TermView view, Function<Term, Term> f) implements TermView {
```

Comment:
Isn't this just `TermMap` but weaker?

Reply:
Yes, this is just `TermPostMap(view, id, f)`.
Not sure if we need this just for convenience.

Reply:
I removed the `TermPreMap` and `TermPostMap` records, but still keep the methods.

Reply:
Great